bot/seasons/halloween/hacktoberstats.py

```diff
@@ -1,17 +1,20 @@
 import json
 import logging
 import re
-import typing
 from collections import Counter
 from datetime import datetime
 from pathlib import Path
+from typing import List, Tuple
 
 import aiohttp
 import discord
 from discord.ext import commands
 
 log = logging.getLogger(__name__)
 
+CURRENT_YEAR = datetime.now().year  # Used to construct GH API query
```

Comment:
If the extension is loaded before the new year, then after the new year it will still show the wrong date until the extension is reloaded. In practice this season will not be loaded during the new year though 🤔

Reply:
Is there a scenario where this wouldn't be loaded at the beginning of October and unloaded at the end?

Reply:
I think only through a season override.